File: 0796-rotate-string/0796-rotate-string.py
class Solution:
    def rotateString(self, s: str, goal: str) -> bool:
        if s==goal:
            return True
        k=0
        i=1
        while k<len(s):
            s=s[i:]+s[:i]
            k+=1
            if s==goal:
                return True
        return False

File: 0796-rotate-string/0796-rotate-string.cpp
class Solution {
public:
    bool rotateString(string s, string goal) {
        if(s==goal){
            return true;
        }
        int k=0;
        int i=1;
        while(k<s.size()){
            s= s.substr(1) + s.substr(0, 1);
            k++;
            if(s==goal){
                return true;
            }
        }
        return false;
    }
};